docs/reference/ingest.asciidoc

```diff
@@ -410,7 +412,8 @@ PUT _ingest/pipeline/my-pipeline
   "processors": [
     {
       "set": {
-        "field": "received",
+        "description": "Set 'event.ingested' to '_ingest.timestamp' value",
```

Comment:
I might say something here about copying the value out of ingest metadata which is not indexed into the document so that the timestamp is indexed?